ilc/build/webpack.js

```diff
@@ -35,6 +35,11 @@ module.exports = {
             header: () => fs.readFileSync(path.resolve(__dirname, '../public/system.js')),
             afterOptimizations: true,
         }),
+        new WrapperPlugin({
+            test: /\.js$/,
+            header: () => fs.readFileSync(path.resolve(__dirname, '../public/patchEvents.js')),
```

Comment:
It is needed to monkey-patch `popstate`, `hashchange` events before single-spa will patch them

Reply:
Because `setupEvents.js` uses dependencies from our bundle
`patchEvents.js` should not use any dependencies because it is not a part of client.js bundle itself and this file does not have access to them